tests/unit/test_log.c

```diff
@@ -61,169 +42,198 @@ static void append_entry(Log *log, int id, const char *value)
 {
     raft_entry_t *e = make_entry(id, value);
 
-    assert_int_equal(LogAppend(log, e), RR_OK);
+    assert(LogAppend(log, e) == RR_OK);
     raft_entry_release(e);
 }
 
-static void test_log_random_access(void **state)
+static void test_log_random_access()
 {
-    Log *log = (Log *) *state;
+    Log log;
```

Comment:
The initialization and teardown sections are one of the advantages of using cmocka.... but in our case, since we don't use any other capability (although we can in other tests, and maybe in the future) I guess we can remove it for now.

Reply:
It's easy to add this if we want. Just a macro to call a function pointer before and after the test. 

```c
test_run(before_fn, test_fn, after_fn) \
    before_fn();
    test_fn();
    after_fn();
```

That's actually what cmocka does. 
Question is do we want it? I personally like when a test case has all the code it runs (no magic). That's why I didn't add it.